### DISCUSSION TITLE
Arrow icon after update to 1.1

### DISCUSSION THREAD

**Discussion by nsirolli** [xournalpp/xournalpp#3249] [General] [answered]
Hi!

I've just updated to 1.1, and when I'm using the pen tool, the icon is a small dot that I barely can locate on the screen.

In 1.0, when the pen tool was selected, the icon was still an arrow, which changed to a dot only after pressing my stylus.

Is it possible to get that arrow back?

Thanks.

**Comment by Technius** [comment, marked as answer]
Which OS are you using?

> In 1.0, when the pen tool was selected, the icon was still an arrow, which changed to a dot only after pressing my stylus.

This is the default behavior if you're using a stylus and the `Pen` device class is assigned to it in the `Input System` tab in the preferences. To get back the old behavior, change the device class to `Mouse`.

**Reply by nsirolli** [reply]
This fixed it. Thanks!!

PS: I'm using Arch

**Comment by rolandlo** [comment]
Also take a look at the configuration options in the View panel of menu `Edit > Preferences`. You won't get the arrow back like that, but you may create a cursor that is easy to locate and more in line with drawing/writing than the arrow.
